content/finalist/ttjc@Saddamshah/index.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Saddam Shah
+type: "finalist"
+name: "Saddam Shah"
+portfolio: "https://saddamshah.netlify.app/"
+github-link: "https://github.com/Saddamshah/"
+linkedin-link: "https://www.linkedin.com/in/saddam-shah/"
+looking-for: "job"
+image-link: "https://avatars0.githubusercontent.com/u/46083623?s=460&u=d2e51ad2754d31669119720791a3f2a38a05b112&v=4" 
```

Comment:
As per previous comments I advised to use correct format i.e "image_link" so please change all the variables as the same format